lib/results_validators/validation_issue.rb

```diff
@@ -3,15 +3,15 @@
 module ResultsValidators
   class ValidationIssue
     attr_reader :kind, :competition_id
-    def initialize(kind, competition_id, message, **message_args)
-      @message = message
+    def initialize(id, kind, competition_id, **message_args)
+      @id = id
       @kind = kind
       @args = message_args
       @competition_id = competition_id
     end
 
     def to_s
-      format(@message, @args)
+      I18n.t("validators.person.#{@id}", @args)
```

Comment:
Have you tested this? From just staring at the code, I feel like it should be `**@args`

Reply:
I hadn't tested it earlier, did it now and yes you are right `**@args` is working. Thanks